apps/wing/src/cli.ts

```diff
@@ -93,7 +93,7 @@ async function main() {
     showGlobalOptions: true,
   });
 
-  program.name("wing").version(PACKAGE_VERSION);
+  program.name("wing").version(PACKAGE_VERSION, "-v, --version, -V", "output the current version");
```

Comment:
I think commander expects all the short flags first

```ts
program.name("wing").version(PACKAGE_VERSION, "-v, -V, --version", "output the current version");
```

Reply:
Thanks, just tested a combination of them, and looks like commander only honours the first 2 in this list (for some reason)...

Reply:
Ah, I see. To be honest I would get rid of `-V` for now and just update the docs to `-v`. `-V` for version is pretty atypical imo

Reply:
Cool PR open, thanks https://github.com/winglang/wing/pull/6841